README.md

```diff
@@ -11,7 +11,22 @@ gem 'redis-rack-cache'
 
 ## Usage
 
-If you are using redis-store with Rails, consider using the [redis-rails gem](https://github.com/redis-store/redis-rails) instead. For standalone usage:
+If you are using redis-store with Rails, consider using the [redis-rails gem](https://github.com/redis-store/redis-rails) instead.
+However, configuration can be done a such:
+
+```ruby
+# config/application.rb
+module MyApplication
+  class Application < Rails::Application
+    config.action_dispatch.rack_cache = {
+      metastore: ::Rack::Cache::EntityStore::Redis.new('redis://localhost:6379/0/metastore', default_ttl: 10.days.to_i),
```

Comment:
rack-cache is set up a little differently from this: https://github.com/rtomayko/rack-cache/blob/master/lib/rack/cache/options.rb#L38-L79

notice how `default_ttl` is on the same level as metastore and entitystore? I think this is what the hash of options needs to look like:

```ruby
config.action_dispatch.rack_cache = {
  metastore: 'redis://localhost:6379/0/rack-metastore',
  entitystore: 'redis://localhost:6379/0/rack-entitystore',
  default_ttl: 15.minutes.from_now
}
```